src/BuildScriptGenerator/Hugo/HugoPlatformInstaller.cs

```diff
@@ -52,6 +58,7 @@ public virtual string GetInstallerScriptSnippet(string version)
 
                 // Write out a sentinel file to indicate downlaod and extraction was successful
                 .AppendLine($"echo > {Path.Combine(versionDirInTemp, SdkStorageConstants.SdkDownloadSentinelFileName)}");
+            InstallGolangToolingAndLanguage(snippet);
```

Comment:
I hope I'm not confused: please feel free to resolve if it is the case. Should we only run `InstallGolangToolingAndLanguage` on the same condition that we `installHugo.sh` above? Are there some cases where we want one and not the other?

Reply:
Just to be clear, in this block, we run installHugo.sh only if the image type is jamstack because installHugo.sh is run in NodePlatform.cs for cli images. However, we install golang irrespective of the image type.